contrib/helm/clair/templates/configmap.yaml

```diff
@@ -13,7 +13,9 @@ data:
         options:
           # PostgreSQL Connection string
           # https://www.postgresql.org/docs/current/static/libpq-connect.html#LIBPQ-CONNSTRING
-          source: "{{ .Values.config.postgresURI }}"
+          # This should be done using secrets or Vault, but for now this will also work
+
+          source: "postgres://{{ .Values.postgresql.postgresUser }}:{{ .Values.postgresql.postgresPassword }}@{{ template "postgresql.fullname" . }}:5432/{{ .Values.postgresql.postgresDatabase }}?sslmode=disable"
```

Comment:
This should check if a postgresURI is explicitly set, if not default to using the postgresql dependency.

Reply:
I'm finding that if I specify postgresURI in my helm values, then the template produces an invalid configmap.

The source option containing the connection string gets merged with the comment above it (I can fix it by removing one of the whitespace chomping dashes).